README.adoc

```diff
@@ -150,10 +147,18 @@ only supported orchestrators are reported.
   * `stop`: Stop the given service in the box.
   * `restart`: Restart the given service in the box.
 +
---
 The supported options for `service` are: Docker, OpenShift.
 
+1. `vagrant service-manager install-cli [service] [--cli-version] [--path]`
++
+Install the client binary for the specified `service`.
++
+The possible options are:
 
+  * `--cli-version`: Download the specified binary version.
```

Comment:
"Download the binary in the specified version, if it exist"

What is the behavior if the version does not exist or wrong? What is the exit code? Does it throw an exception? What is the current behavior and what do we want it to do? I would think it returns a message of the form "Specified client binary version does not exist" and a non 0 return code. Is this what's happening?